staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -210,6 +210,8 @@ type VirtualMachineInstanceStatus struct {
 	MigrationMethod VirtualMachineInstanceMigrationMethod `json:"migrationMethod,omitempty"`
 	// This represents the migration transport
 	MigrationTransport VirtualMachineInstanceMigrationTransport `json:"migrationTransport,omitempty"`
+	// Reprensents the size of the cloud-init isos associated with the VMI
+	IsoSizes VirtualMachineInstanceIsoSizes `json:"isoSizes,omitempty"`
```

Comment:
How about adding this info to `VolumeStatus`? The structure seems to be close enough to allow reusing it. I just want to avoid cumulating all kinds of "anonymous" maps. @awels would it be possible to extend this struct with the iso size (optional)?

Reply:
Will do!

Reply:
Pushed a commit that moves the size to the volume status.